src/components/edit/Block.tsx

```diff
@@ -31,44 +30,32 @@ const chordSymbolClassName = "ChordSymbol";
 const blockChordSymbolClassName = "BlockChordSymbol";
 const blockChordTargetClassName = "BlockChordTarget";
 
-const ChordInput = withStyles((theme: Theme) => ({
-    root: {
+// this height keeps the chords, the chord input, and lyrics the same height
+// so that everything lines up
+const rowHeight = "2em";
```

Comment:
this was an unexpected change that was needed - the heights didn't line up for some reason and that's because they only lined up by coincidence and it broke in the migration

setting the row height made things look good again. without doing this, the chords and lyrics would shift heights during chord editing